### PR TITLE
Add CI with GitHub Actions

### PR DESCRIPTION
**Pull request by matthewfeickert**
Add skeleton of CI with GitHub Actions to be filled in with pytest tests later.

Also fix some Python 3 issues and add missing dependencies.

### CODE REVIEW

**Comment by matthewfeickert** [comment]
@lukasheinrich This is ready for review.

File: pylhe/__init__.py [hunk 1]
```diff
@@ -133,8 +134,8 @@ def readLHEInit(thefile):
                 if child.tag == "weightgroup" and child.attrib != {}:
                     try:
                         wg_type = child.attrib["type"]
-                    except Exception, e:
-                        print "weightgroup must have attribute 'type'"
+                    except Exception:
+                        print("weightgroup must have attribute 'type'")
                         sys.exit(1)
```
Comment (lukasheinrich) [inline pylhe/__init__.py]: can we just re-raise here instead of exiting?

File: pylhe/__init__.py [hunk 2]
```diff
@@ -144,8 +145,8 @@ def readLHEInit(thefile):
                             continue
                         try:
                             wg_id = w.attrib["id"]
-                        except Exception, e:
-                            print "weight must have attribute 'id'"
+                        except Exception:
+                            print("weight must have attribute 'id'")
                             sys.exit(1)
```
Comment (lukasheinrich) [inline pylhe/__init__.py]: dito

File: pylhe/__init__.py (hunk 1 above)
Comment (lukasheinrich) [reply pylhe/__init__.py]: probably should be KeyError

Comment (matthewfeickert) [reply pylhe/__init__.py]: Yes, definitely. These are just the things that were in before from the other PR (with a quick edit to get pyflakes passing).

Comment (matthewfeickert) [reply pylhe/__init__.py]: Updated.

**Comment by lukasheinrich** [comment]
awesome.. thanks @matthewfeickert!

**Pull request merged**